src/SFML/Window/OSX/SFOpenGLView+mouse.mm

```diff
@@ -384,8 +384,8 @@ - (NSPoint)cursorPositionFromEvent:(NSEvent*)eventOrNil
         NSSize  size   = [self frame].size;
         NSPoint origin = [self frame].origin;
         NSPoint oldPos = rawPos;
-        rawPos.x       = std::min(std::max(origin.x, rawPos.x), origin.x + size.width - 1);
-        rawPos.y       = std::min(std::max(origin.y + 1, rawPos.y), origin.y + size.height);
+        rawPos.x       = std::clamp(origin.x, rawPos.x, origin.x + size.width - 1);
+        rawPos.y       = std::clamp(origin.y + 1, rawPos.y, origin.y + size.height);
```

Comment:
It makes more sense this way:
```suggestion
        rawPos.x       = std::clamp(rawPos.x, origin.x, origin.x + size.width - 1);
        rawPos.y       = std::clamp(rawPos.y, origin.y + 1, origin.y + size.height);
```
In addition, the behavior of clamp is undefined if the lower bound is greater than the upper bound, which could happen here.

Reply:
Oh yeah I see how the previous min/max implementation meant you could mix up the ordering. Great catch.